hypothesis-python/tests/cover/test_lookup.py

```diff
@@ -962,14 +980,16 @@ def test_from_type_can_be_default_or_annotation():
 
 @pytest.mark.parametrize("t", BUILTIN_TYPES, ids=lambda t: t.__name__)
 def test_resolves_builtin_types(t):
-    v = st.from_type(t).example()
-    assert isinstance(v, t)
+    with pytest.warns():  # some types may warn SmallSearchSpaceWarning
```

Comment:
Do you want to assert that there is a warning, or ignore one if it happens?

Comment implies the latter, for which the stdlib catch_warnings context manager is more appropriate.